ecolex/legislation.py

```diff
@@ -196,14 +200,21 @@ def harvest_file(upfile):
                 legDate = datetime.strptime(legYear, '%Y')
                 legislation['legDate'] = legDate.strftime('%Y-%m-%dT%H:%M:%SZ')
             except Exception:
-                logger.debug('Error parsing legYear %s' % (legYear))
-
-        url_value = document.attrs.get('url', None)
-        if url_value:
-            legislation['legLinkToFullText'] = url_value
+                logger.debug(f'Error parsing legYear {legYear}')
+
+        filenames = get_content(document.findAll('link_to_full_text'))
+        url_values = []
+        for filename in filenames:
+            if filename.endswith('.doc'):
+                path = settings.DOC_URL
+            elif filename.endswith('.html'):
+                path = settings.HTML_URL
+            else:
+                path = settings.PDF_URL
```

Comment:
Is there any chance that the extension is not included in this list `['doc', 'html', 'pdf']`?
We could save the three urls in settings under a dictionary:
```
FULL_TEXT_URLS = {
    'doc': DOC_URL,
    'html': HTML_URL,
    'pdf': PDF_URL,
}
```

Then the rest of the code can be compacted like this & it's easier to modify if new extensions are added:
```
extension = filename.rsplit('.')[-1]
url = settings.FULL_TEXT_URLS[extension]
url_values.append(f'{url}{filename}')
```